src/jit/codegenlegacy.cpp

```diff
@@ -18830,15 +18830,17 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
                                                    (ssize_t)stubAddr);
                             // The stub will write-back to this register, so don't track it
                             regTracker.rsTrackRegTrash(compiler->virtualStubParamInfo->GetReg());
-                            getEmitter()->emitIns_R_R_I(INS_ldr, EA_PTRSIZE, REG_JUMP_THUNK_PARAM,
```

Comment:
I thought we were required to use R12 for this because the VM stubs required it.

I don't think you should change the non-R2R code path.